app/controllers/crypto_address_for_channels_controller.rb

```diff
@@ -78,6 +78,26 @@ def change_address
     end
   end
 
+  def delete
+    chain = params[:chain]
+    current_channel = current_publisher.channels.find(params[:channel_id])
+
+    existing_address = CryptoAddressForChannel.where(chain: chain, channel: current_channel)
```

Comment:
This can be cleaned up to 
`CryptoAddressForChannel.where(chain: chain, channel: current_channel).first&.destroy!`